lib/Raven/Stacktrace.php

```diff
@@ -23,13 +23,11 @@ class Stacktrace implements \JsonSerializable
      */
     const CONTEXT_NUM_LINES = 5;
 
-    /**
-     * @var Client The Raven client
-     */
-    protected $client;
+    /** @var Configuration */
```

Comment:
This is missing the description of the variable and is not following how all other docblocks are organized (multiline)